packages/xo-server/src/api/vm.js

```diff
@@ -621,6 +621,9 @@ set.params = {
 
   // set the VM network interface controller
   nicType: { type: ['string', 'null'], optional: true },
+
+  // set the VM boot firmware mode
+  hvmBootFirmware: { type: ['string', 'null'], optional: true },
```

Comment:
Should be also in `vm.create`.

Reply:
I add it. But it's weird why I didn't get `invalid params` error when I called `vm.create` with `hvmBootFirmware` param included in `params` !
Does schema-inspector inspect only inspected params ?

Reply:
Until last Friday (b8524732ce5fc466ef73664ff02cd4e09fca7ad9), we did not throw on extraneous params.

It should now be checked correctly.

Reply:
Thank you for clarifying me this :)